### PR TITLE
ISO website with react

### PR DESCRIPTION
**Pull request by pocky**
Hello!

Same website but with react. I'm not sure about deployment and gh-pages but... We will see ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2444 @@
-# LePetitBloc.github.io
-LePetitBloc Website
```
Comment (lutangar) [inline README.md]: We could keep the old README and even add a bit more to it.

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (lutangar) [inline public/manifest.json]: The manifest need to be changed.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>LePetitBloc - Startup studio for blockchains projects</title>
+    <meta name="description" content="If you love Blockchain and software craftsmanship: we are here."/>
```
Comment (lutangar) [inline public/index.html]: Something closer to 150 chars would be better for the description.
BTW for a few basics, this tool is quite useful: https://frontendchecklist.io/

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+        <div className="App">
```
Comment (lutangar) [inline src/App.js]: You can have an empty tag here if it's what you wanted: 
```
<>
....
</>
```
Is a valid synthax.

Comment (pocky) [reply src/App.js]: "Le JS c'est du caca"  ©️React documentation on fragments with Babel.

**Pull request merged**